hack/install.sh

```diff
@@ -124,6 +124,37 @@ do_install() {
 	if [ -z "$lsb_dist" ] && [ -r /etc/lsb-release ]; then
 		lsb_dist="$(. /etc/lsb-release && echo "$DISTRIB_ID")"
 		dist_version="$(. /etc/lsb-release && echo "$DISTRIB_CODENAME")"
+		if [ -z "$dist_version" ]; then
+			dist_version="$(. /etc/lsb-release && echo "$DISTRIB_RELEASE")"
+			case "$lsb_dist" in
+				"Debian")
+					case "$dist_version" in
+						8)
+							dist_version="jessie"
+							;;
+						7)
+							dist_version="wheezy"
+							;;
+					esac
+					;;
+				"Ubuntu")
+					case "$dist_version" in
+						"15.04")
+							dist_version="vivid"
+							;;
+						"14.04")
+							dist_version="trusty"
+							;;
+						"13.10")
```

Comment:
So I'll update the doc as well cause I found it somewhere in the doc yesterday :/